app/pods/publications/publication/translations/documents/controller.js

```diff
@@ -181,19 +188,17 @@ export default class PublicationsPublicationTranslationsDocumentController exten
   }
 
   @action
-  isDeleteDisabled(piece) {
-    return this.translationSubcase.isFinished
-      // can be translation or publication related
-      || piece.requestActivitiesUsedBy.length > 0;
+  isPieceDeletable(piece) {
```

Comment:
Kunnen we in `translations/documents` dan ook niet beter met een `PieceRow` class werken zoals we nu in `proofs/documents` doen en deze actie als een get-function definiëren op die class? Dan werken de 2 routes analoog.

Reply:
@ValenberghsSven was hier van mening dat het misschien beter is m dit in een nieuw ticket te herwerken.
Als ik hem juist verstaan had.

Reply:
Leek mij idd buiten scope + deze PR staat nu al redelijk lang open

Reply:
> de `isOvrb` toevoegingen kan ik niet zomaar zeggen wat daar de beste manier is.
> Hangt ervan af wat we hiermee willen bereiken en welke profielen toegang hebben tot publicaties.
> Het was ook niet expliciet gevraagd in de story, laten we dat dan er voorlopig uit?
> De delete afschermen ben ik sowieso voor, maar andere profielen (kabinet, minister, overheid) zien geen publicaties (geen propagatie) en hebben geen rechten om deletes uit te voeren.
> De code nu zorgt er voor dat `kanselarij`geen deletes kan doen, maar ze kunnen wel andere acties uitvoeren (zoals vertalingen uploaden, aanvragen starten etc..).
> Ik wil hier eerder aangeven dat het gebruik van `isOvrb` nu inconsistent is, en mogelijk doen we dit beter in een ander ticket

De `isOvrb`-check is woensdag besproken in de demo. Conclusie was dat iedereen die toegang heeft tot de publicaties ook alle acties moet kunnen doen. Er wordt nog nagevraagd wie allemaal toegang moet hebben tot de root-publications route.
Ik heb de bijkomende checks op `isOvrb` in de publications route/components eruit gehaald. Access tot de root-route wordt verder opgenomen in een vervolgstory.